### DISCUSSION TITLE
Github maintence schedule time

### DISCUSSION THREAD

**Discussion by Prabha-56** [community/community#128510] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Team, i see github maintenance notification. pls let us know by when we expect github available back.?

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Prabha-56 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
